hw/top_earlgrey/util/opentitan_earlgrey_usbdev_expected-uart

```diff
@@ -2,8 +2,8 @@ I00000 boot_rom.c:52] Version:    opentitan-snapshot-20191101-1-3257-g1690ca0
 Build Date: 2020-11-23, 17:36:40
 
 I00001 boot_rom.c:61] Boot ROM initialisation has completed, jump into flash!
-I00000 hello_usbdev.c:145] Hello, USB!
-I00001 hello_usbdev.c:146] Built at: Nov 23 2020, 17:36:42
+I00000 hello_usbdev.c:146] Hello, USB!
+I00001 hello_usbdev.c:147] Built at: Nov 27 2020, 14:21:54
 I00002 demos.c:18] Watch the LEDs!
-I00003 hello_usbdev.c:157] PHY settings: pinflip=0 differential=0
-Hi!Hi!I00004 hello_usbdev.c:205] PASS!
+I00003 hello_usbdev.c:159] PHY settings: pinflip=0 differential=0 TI Phy=0

```

Comment:
The format of these log lines should *not* be used in tests. There is a high chance that I will break this test when performing refactors of the core libraries. Instead, you should probably be searching for the specific log message (without the prefix) you want to check for.

Reply:
Good point @mcy. We could (and probably should) transform these tests into system tests (living in `util/systemtest`) where we have the necessary framework and helpers to strip log messages of such prefixes. 

I can have a look at that if you think that's a good idea @mdhayter, and I'm fine leaving the tests in the fragile state they are currently until then.

(@mcy: We have unfortunately way too little manpower on the USB components and I'd like to avoid a situation where me make better the enemy of good.)